src/game/Tactical/PathAI.cc

```diff
@@ -719,13 +721,13 @@ INT32 FindBestPath(SOLDIERTYPE* s, INT16 sDestination, INT8 ubLevel, INT16 usMov
 	queRequests = 2;
 
 	//initialize the path data structures
-	memset( pathQ, 0, iMaxPathQ * sizeof( path_t ) );
-	memset( trailTree, 0, iMaxTrailTree * sizeof( trail_t ) );
+	std::fill_n(pathQ, iMaxPathQ, path_t{});
+	std::fill_n(trailTree, iMaxTrailTree, trail_t{});
 
 #if defined( PATHAI_VISIBLE_DEBUG )
 	if (gfDisplayCoverValues && gfDrawPathPoints)
 	{
-		memset( gsCoverValue, 0x7F, sizeof( INT16 ) * WORLD_MAX );
+		std::fill_n(gsCoverValue, WORLD_MAX, 0x7F7F);
```

Comment:
What does `0x7F` actually stand for in this case?

Reply:
It's a magic debug number.